src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -575,11 +576,12 @@ def run_acr_build(self, dockerfile, source, quiet=False, build_from_source=False
                 # Temporary fix: using run time tag as customer image tag
                 # Waiting for buildpacks side to fix this issue: https://github.com/buildpacks/pack/issues/1750
                 logger.warning("Attempting to build image using buildpacks...")
+                buildpack_image_name_with_tag = image_name_with_tag
                 run_image_tag = get_latest_buildpack_run_tag("aspnet", "7.0")
                 if run_image_tag is not None:
-                    image_name = f"{image_name}:{run_image_tag}-{tag_now_suffix}"
-                self.build_container_from_source_with_buildpack(image_name, source)
-                self.image = self.registry_server + "/" + image_name
+                    buildpack_image_name_with_tag = f"{image_name}:{run_image_tag}-{tag_now_suffix}"
+                self.build_container_from_source_with_buildpack(buildpack_image_name_with_tag, source)
+                self.image = self.registry_server + "/" + buildpack_image_name_with_tag
```

Comment:
check indent. Should self.build_container_from_source_with_buildpack and self.image in the same part?

Reply:
Thanks for your comment! As discussed on Teams, we confirm the indent looks good.